celery/signals.py

```diff
@@ -105,6 +106,7 @@
 # - Worker
 import_modules = Signal(name='import_modules')
 worker_init = Signal(name='worker_init')
+worker_before_create_process = Signal(name="worker_before_create_process")
```

Comment:
would like to have some unit/integration tests to see its working well and won't create any regression

Reply:
+1